File: c/03-estrutura-repetitiva/tabuada.c
/*Problema "tabuada"
Ler um número inteiro N, daí mostrar na tela a tabuada de N para 1 a 10, conforme exemplo.*/


#include <stdio.h>
int main () {
    int n; 

    printf("Deseja a tabuada para qual valor? ");
    scanf("%d", &n);

    for(int cont = 1; cont <= 10; cont ++) {       
        printf("%d x %d = %d\n", n, cont, n * cont);
    }
    return 0;
}

File: java/03-estruturas-repetitivas/Tabuada.java
/*Problema "tabuada"
Ler um número inteiro N, daí mostrar na tela a tabuada de N para 1 a 10, conforme exemplo.*/

import java.util.Locale;
import java.util.Scanner;

public class Tabuada {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int n;
		
		System.out.print("Deseja a tabuada para qual valor? ");
		n = sc.nextInt();
		
		for(int i = 1; i <= 10; i++) {
			System.out.println(n + " x " + i + " = " + n * i);
		}
		
		sc.close();
	}

}
